internal/runtime/helper.go

```diff
@@ -30,6 +33,51 @@ func constructTransactions(
 		nonceMap = make(map[uint64]uint64) // accountNumber -> nonce
 	)
 
+	fmt.Printf("\n⏳ Estimating Gas ⏳\n")
+
+	// Estimate the fee for the transaction batch
+	txFee := defaultDeployTxFee
+
+	// Construct the first tx
+	// Generate the transaction
```

Comment:
Is one of these two comments in the wrong place?

Reply:
Dropped

[5384516](https://github.com/gnolang/supernova/pull/66/commits/5384516f764995b8caf4e9485683907480ab183b)